docs/rules/key-spacing.md

```diff
@@ -67,7 +67,7 @@ foo = { thisLineWouldBeTooLong:
 
 ### 2. Vertically align values `"align": "value"`
 
-Use the `align` option to enforce vertical alignment of values in an object literal. This mode still respects `beforeColon` and `afterColon` where possible, but it will pad with spaces after the colon where necessary. Groups of properties separated by blank lines are considered distinct and can have different alignment than other groups.
+Use the `align` option to enforce vertical alignment of values in an object literal. This mode still respects `beforeColon` and `afterColon` where possible, but it will pad with spaces after the colon where necessary. Groups of properties separated by blank lines are considered distinct and can have different alignment than other groups. Single lines of code will not be checked for vertical alignment, but will still be checked for `beforeColon` and `afterColon`.
```

Comment:
Replacing "Single lines of code" with "Single-line object literals" or something similar would make this less ambiguous.